buildroot-external/board/raspberrypi/uboot-boot64.ush

```diff
@@ -18,7 +18,7 @@ test -n "${BOOT_A_LEFT}" || setenv BOOT_A_LEFT 3
 test -n "${BOOT_B_LEFT}" || setenv BOOT_B_LEFT 3
 
 # HassOS bootargs
-setenv bootargs_hassos "zram.enabled=1 zram.num_devices=3 apparmor=1 security=apparmor rootwait systemd.machine_id=${MACHINE_ID} cgroup_enable=memory"
+setenv bootargs_hassos "zram.enabled=1 zram.num_devices=3 apparmor=1 security=apparmor rootwait systemd.machine_id=${MACHINE_ID} cgroup_enable=memory fsck.repair=yes"
```

Comment:
We should do that on all platforms. Default is `preen` which is safer to run in theories, but maybe it helps on some issues which I saw on Sentry.

Reply:
Is there a central place to add bootargs for all platforms?

Reply:
Or we can start with the default of preen for now. I think the bigger change is calling systemd-fsck on each mount, and even preen should fix most issues.

Reply:
There is no central place, you need but it on each uboot script, but that is not much place. `preen` is already running in the background after mounting. So it would make sense to go over all uboot scripts or search for `bootargs_hassos` and but it in. 30s with vscode :)

Reply:
>preen is already running in the background after mounting.

Are you sure about this? I thought fsck is not being run at all right now because we don't have any systemd-fsck dependencies setup. By default only systemd-fsck-root runs.

Reply:
>https://www.freedesktop.org/software/systemd/man/systemd-fsck@.service.html#Kernel%20Command%20Line

Like I understand is the mode auto with repair type preen the default value if you don't set one.

Reply:
Yes auto and preen are defaults, but they only kick in for entries inside fstab:

>These services are started at boot if passno in /etc/fstab for the file system is set to a value greater than zero.

Since our fstab is empty, systemd-fsck is never run so it never looks at the preen option or runs fsck: https://github.com/systemd/systemd/blob/master/src/fsck/fsck.c#L96-L97

Only with the systemd dependencies added in the rest of this PR will preen mode run. So that's why I'm wondering maybe we only keep it as preen for now and if there are more problems in the future we can switch it to forced yes.

Reply:
BTW, `buildroot/board/hardkernel/odroidxu4/boot.ini` also has `fsck.repair=yes`. Not sure why.